upgrades/upgrade-1.6-2.0.rst

```diff
@@ -0,0 +1,291 @@
+Upgrading from Sulu 1.6 to 2.0
+==============================
+
+We have a very extensive `UPGRADE.md file`_ in our repository, which contains almost every breaking change we made
+during the development of Sulu 2.0. However, this is a huge list and most projects won't be affected by the majority of
+these breaking changes. Therefore we are going through the most important changes in this guide, and by following these
+steps you can probably upgrade most of the projects, if you have not overridden some internal mechanisms.
+
+**1. Create a new** `sulu/skeleton`_ **project and import your projects** ``.git`` **folder**
+
+In our experience it makes more sense and will save you future headache if you start over with a new project, because
+`sulu/skeleton`_ is the new way starting a Sulu 2.0 project and running a `sulu/sulu-minimal`_ or `sulu/sulu-standard`_
+project using Sulu 2.0 is not supported. You could also take your existing `sulu/sulu-minimal`_ project and restructure
+it to match the `sulu/skeleton`_ repository, but that is probably going to be more work than doing it the other way
+around.
+
+The recommended way to achieve this, is to create a new project using ``composer create-project`` and copy the old
+``.git`` folder to the root of the new project.
+
+.. code-block:: bash
+
+    composer create-project sulu/skeleton <project-name>
+    cp <old-project-path>/.git <project-name>
+
+Afterwards a ``git status`` in the new folder should reveal all the changes from your old repository to the new
+`sulu/skeleton`_ installation, since ``git`` is doing a very good job on recognizing the file changes here.
+
+.. note::
+    Theoretically these changes could already be commited now, but it is at the end of this guide, so that you have a
```

Comment:
but we recommend to do this at the end of this guide. That will create a single commit that contains the whole upgrade

Reply:
Rephrased it a bit to avoid having `that` two times